c7n/resources/cw.py

```diff
@@ -243,7 +288,7 @@ class EventRule(QueryResourceManager):
 
     class resource_type(TypeInfo):
         service = 'events'
-        arn_type = 'rule'
+        arn = 'Arn'
```

Comment:
this effectively also needs to change rules to being a child resource

Reply:
I figured out how this should work. Will update PR soon. Thanks.